src/test/scala/co/ledger/template/repository/UserRepositorySpec.scala

```diff
@@ -36,6 +36,9 @@ class UserRepositorySpec extends RepositorySpec {
     }
   }
 
+  test("find all users query") {
+    check(UserStatement.findAll())
+  }
```

Comment:
Not sure to understand this very well :)

I also wanted to add a test similar to the "Find a user" test, but for "Find all users". But I was always ending up with a list of just one item. i guess I expected 3 items (all the test users). But I guess that's not how it works 🤷